api/controllers/swagger.go

```diff
@@ -316,6 +317,24 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 					},
 				},
 			},
+			"/task/{id}/logs": map[string]swagger.Method{
+				"get": {
+					Summary: "Get task logs",
+					Tags:    []string{"Task"},
+					Parameters: []swagger.Parameter{
+						swagger.NewStringPathParam("id", "ID of the task to describe", true),
+						swagger.NewIntQueryParam("tail", "The number of lines from the end to return", false),
+						swagger.NewStringQueryParam("start", "The start of the time range to fetch logs (format YYYY-MM-DD HH:MM)", false),
+						swagger.NewStringQueryParam("end", "The end of the time range to fetch logs (format YYYY-MM-DD HH:MM)", false),
+					},
+					Responses: map[string]swagger.Response{
+						"200": {
+							Description: "The task's logs",
```

Comment:
[nit] should be `tasks'`

Reply:
I don't think it is actually - it's the logs belonging to a single task (as opposed to the containers' logs)

Reply:
gotcha, disregard